roles/ec2_provision/files/nfs_sg.yml

```diff
@@ -0,0 +1,38 @@
+- name: Create NFS instance security group
```

Comment:
If these are tasks to be selectively imported , then they should be stored in the role/tasks directory just like the rest of tasks. I think files/ purpose is to actually hold scripts (bash, python) and other similar tools.

Reply:
At the moment it is not clear how different will be every instance provisioning. I agree with you at this point, will adjust this one after. But considering a wide variety of possible instance setups, it is possible to have a lot of similar tasks at the role directory, which will not be as comprehensive.

Reply:
On the second look, for the moment it makes more sense, will be fixed.